uptane-standard.md

```diff
@@ -868,32 +873,37 @@ In order to perform partial verification, an ECU SHALL perform the following ste
-5. Download and check the Snapshot metadata file from the Director repository, following the procedure in {{check_snapshot}}.
-6. Download and check the Targets metadata file from the Director repository, following the procedure in {{check_targets}}.
-7. Download and check the Root metadata file from the Image repository, following the procedure in {{check_root}}.
-8. Download and check the Timestamp metadata file from the Image repository, following the procedure in {{check_timestamp}}.
-9. Download and check the Snapshot metadata file from the Image repository, following the procedure in {{check_snapshot}}.
-10. Download and check the top-level Targets metadata file from the Image repository, following the procedure in {{check_targets}}.
-11. Verify that Targets metadata from the Director and Image repositories match. A primary ECU MUST perform this check on metadata for all images listed in the Targets metadata file from the Director repository downloaded in step 6. A secondary ECU MAY elect to perform this check only on the metadata for the image it will install. (That is, the target metadata from the Director that contains the ECU identifier of the current ECU.) To check that the metadata for an image matches, complete the following procedure:
-    1. Locate and download a Targets metadata file from the Image repository that contains an image with exactly the same file name listed in the Director metadata, following the procedure in {{resolve_delegations}}.
-    2. Check that the Targets metadata from the Image repository matches the Targets metadata from the Director repository:
-        1. Check that the non-custom metadata (i.e., length and hashes) of the unencrypted image are the same in both sets of metadata.
-        2. Check that all "MUST match" custom metadata (e.g., hardware identifier and release counter) are the same in both sets of metadata.
-        3. Check that the release counter in the previous targets metadata file is less than or equal to the release counter in this targets metadata file.
+1. Load the repository mapping metadata ({{repo_mapping_meta}}).
+2. Load the latest verified time from the time server, if implemented.
+3. Refresh the top-level metadata for the Director repository, by following the steps in {#refresh_toplevel_metadata}, for the first repository in the list of repositories in the mapping metadata.
+4. If this ECU is a primary ECU, then iterate through the image metadata listed in the Director repository's top-level Targets metadata, and **store** all image metadata.  The primary ECU SHOULD also ensure that the image metadata from the director repository doesn't contain any ECU identifiers for ECUs not actually present in the vehicle.
+5. If instead this ECU is a secondary ECU, then iterate through the image metadata listed in the Director repository's top-level Targets metadata, and **store** the first image metadata listed that includes this secondary ECU's ECU identifier and hardware ID.
```

Comment:
Why the first image metadata? I know this is to ignore duplicates, but still might be good to clarify. Also, where to store this image metadata?

Reply:
Now that you ask that, it occurs to me that we don't require this list to be ordered (and indeed shouldn't IMO).  The intention I had when I wrote "first" there was to keep things deterministic in case of duplicates (which may have, say, different versions of the image with different hardware IDs or release counters, etc.).

But that kind of management should probably be left to the implementer.  So -- I'm for removing "first" and otherwise leaving it as is, I think.  If anyone has any other opinions....?